templates/6_tma_batch_qc_metrics.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<p>This should be a more generic placeholder instead of TONIC</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>